sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -84,9 +84,6 @@ public void LogEvent(EventWrittenEventArgs args)
                 }
             }
 
-            // Dump the message and category
-            Trace.WriteLine(message, category);
```

Comment:
Do we still keep the error output?

Reply:
Hmm do you know of a pipeline that has flaky tests? 😆

Reply:
ServiceBus? :)

Reply:
Heh, well we don't have recorded tests so the logging isn't enabled.

Reply:
I will test locally at least.

Reply:
Well it's still outputting logs for all tests in CI.

Reply:
Let me try reverting to the earlier NUnit.